Day-I-Social-Card/social-card/public/index.html

```diff
@@ -37,4 +37,9 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
   </body>
+  <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
```

Comment:
you don't need this here, you have these packages as dependencies in your package.json, so they get installed locally when you do `yarn install`